RELEASING.md

```diff
@@ -4,11 +4,11 @@ Release procedure for dependency-analysis-android-gradle-plugin
 1. Bump version number to next stable version (use semantic versioning: x.y.z)
 1. Update README if needed
 1. git commit -am "Prepare for release x.y.z." 
-1. git tag -a vx.y.z -m "Version x.y.z"
+1. git tag -a x.y.z -m "Version x.y.z"
 1. Execute `./gradlew publishPlugins` to release to the [Plugin Portal](https://plugins.gradle.org/plugin/com.autonomousapps.dependency-analysis)
 This will also run the tests.
-1. Update `build.gradle.kts` to next snapshot version (x.y.z-SNAPSHOT)
-1. Bump `latestRelease` to stable version just published.
+1. Update `build.gradle.kts` to next snapshot version (x.y.z+1-SNAPSHOT)
```

Comment:
What's the convention you're proposing here? What will iterative snapshots look like?

Reply:
There is no special convention, x.y.z was released in the previous step, so the next snapshot cannot be named x.y.z-snapshot, it's an increased version number, and the simplest way to express it is to increase z. If it's not clear the remove the whole parentheses.

Reply:
For the '+1', it's just about saying that we should not forget that while adding `-SNAPSHOT`, we should also not forget to increase the version number. Let me know if you find a better / more clear way to say it.